install.sh

```diff
@@ -0,0 +1,34 @@
+main() {
+
+	if which tput >/dev/null 2>&1; then
+      ncolors=$(tput colors)
+  fi
+  if [ -t 1 ] && [ -n "$ncolors" ] && [ "$ncolors" -ge 8 ]; then
+    RED="$(tput setaf 1)"
+    GREEN="$(tput setaf 2)"
+    NORMAL="$(tput sgr0)"
+  else
+    RED=""
+    GREEN=""
+    NORMAL=""
+  fi
+
+  set -e
+
+  echo "${GREEN}Installing DaVinci Video Converter..."
```

Comment:
Please don't use the wording "DaVinci Video Converter..." It suggests that this is a tool from Blackmagicdesign / DaVinci Resolve, which is not true. 

Use "convert-footage" instead.